src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,108 @@
+use liquid;
+use std::borrow::Cow;
+use std::vec::Vec;
+use super::SortOrder;
+
+// struct Trails {
+//   before: i32,
+//   after: i32,
+// }
+
+#[derive(Clone, Debug)]
+pub(crate) struct PaginationCfg {
+  pub(crate) include: String,
+  pub(crate) per_page: i32,
+  pub(crate) permalink: String,
+  pub(crate) order: SortOrder,
+  pub(crate) sort_by: Vec<String>,
+  pub(crate) post_include: Option<String>
```

Comment:
Whats the difference between an `include` and a `post_include`?

Reply:
`post_include` shouldn't been there, it's a left over from my attempt to `tags` pagination. I needed it to add the tag value after the `include` in the permalink